README.md

```diff
@@ -95,6 +100,33 @@ iex> Litmus.validate(params, schema)
 {:error, "new_user must be a boolean"}
 ```
 
+### Litmus.Type.List
+
+The `List` module contains options that will validate List data types. It supports the following options:
+  * `:min_length` - Specifies the minimum list length. Allowed values are non-negative integers.
+  * `:max_length` - Specifies the maximum list length. Allowed values are non-negative integers.
+  * `:length` - Specifies the exact list length. Allowed values are non-negative integers.
```

Comment:
It looks like `0` is not supported. Do you want to support it, or change this to `positive integers`.

Reply:
We are supporting 0. Did you find a case where we are not?